index.ch.md

```diff
@@ -0,0 +1,17 @@
+_2021-03-23_
+
+Richard M. Stallman，经常被称为RMS，长期以来一直是自由软件社区的一股危险力量。他已经证明了自己是一个厌恶女性、残疾主义者和跨性别者，以及其他一些不当行为的严重指控。这些信仰在自由软件、数字权利和技术社区中没有立足之地。随着他最近重新回到自由软件基金会董事会，我们呼吁FSF的整个董事会下台，并要求RMS从所有的领导职位上被移除。
```

Comment:
> 厌恶女性、残疾主义者和跨性别者
Are you translating this using google translate or something? That's not the right words at all. In fact they have opposite meaning.